src/api/messageprocessors/ListingItemMessageProcessor.ts

```diff
@@ -37,19 +37,14 @@ export class ListingItemMessageProcessor implements MessageProcessorInterface {
         // get Category
         const itemCategoryId = await this.createUserDefinedItemCategories(data.information.category);
         data.information.itemCategory = itemCategoryId;
```

Comment:
how do you know that the categories passed in are "UserDefined"? you don't, so the naming is wrong, and it makes this code unreadable.